src/map_object.c

```diff
@@ -701,36 +691,36 @@ BOOL sub_0205EEF4(MapObjectManager* manager, LocalMapObject** object_dest, int*
     return FALSE;
 }
 
-void sub_0205EF48(LocalMapObject* object) {
+static void sub_0205EF48(LocalMapObject *object) {
     MapObject_SetFlagsBits(object, MAPOBJECTFLAG_UNK2);
     sub_0205EF5C(object);
 }
 
-void sub_0205EF5C(LocalMapObject* object) {
-    MapObject_ClearFlagsBits(object, MAPOBJECTFLAG_UNK26 |
-                                     MAPOBJECTFLAG_UNK24 |
-                                     MAPOBJECTFLAG_UNK20 |
-                                     MAPOBJECTFLAG_UNK15);
+static void sub_0205EF5C(LocalMapObject *object) {
+    MapObject_ClearFlagsBits(object, (MapObjectFlagBits)(MAPOBJECTFLAG_UNK26 | MAPOBJECTFLAG_UNK24 |
+                                                         MAPOBJECTFLAG_UNK20 | MAPOBJECTFLAG_UNK15));
 }
 
-void sub_0205EF6C(LocalMapObject* object) {
-    // No-op
+static void MapObject_NoOp(LocalMapObject *object) {
+    // NoOp
 }
 
-u32 ResolveObjectGfxId(FieldSystem* fieldSystem, int a1) {
-    if (a1 >= 101 && a1 <= 117) {
-        a1 = FieldSystem_VarGetObjectEventGraphicsId(fieldSystem, (u16)(a1 - 101));
+static u32 FieldSystem_ResolveObjectSpriteID(FieldSystem *fieldSystem, s32 spriteId) {
+    //BUG: index out of range, should be < not <=
+    //TODO: check these with sprite constants, these should be variable sprites but this starts at 101 not 100
```

Comment:
do TODO?

Reply:
this would require more sprite code decomp/extraction to check, so it's a problem for future me when that has been done